core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/TelemetryType.java

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.applicationinsights.internal.channel.samplingV2;
```

Comment:
See comment in FixedRateSamplingTelemetryProcessor. This should be removed.

Reply:
See my response above for it.

Reply:
Will be amended based on my response above.

Reply:
#resolved